dask/dataframe/core.py

```diff
@@ -482,12 +482,13 @@ def get_partition(self, n):
             raise ValueError(msg)
 
     @derived_from(pd.DataFrame)
-    def drop_duplicates(self, split_every=None, split_out=1, **kwargs):
+    def drop_duplicates(self, subset=None, split_every=None, split_out=1, **kwargs):
         # Let pandas error on bad inputs
         self._meta_nonempty.drop_duplicates(**kwargs)
```

Comment:
@WesRoach Should this call include `subset` parameter? In case of non-existing column(s) provided in subset parameter this needs to error out?

Reply:
Yeah, you need to add the `subset` parameter here.

```
self._meta_nonempty.drop_duplicates(subset=subset, **kwargs)
```

Reply:
This `drop_duplicates` is used by both DataFrame and Series. If this `self._meta_nonempty.drop_duplicates(subset=subset, **kwargs)` was added it would error out for series as `Series.drop_duplicates` doesn't accept subset param.
So check if subset present, then call drop_duplicates with subset else call without it.

Reply:
Thanks for feedback! Let me know if this looks correct. New to project - feedback is very welcome.